packages/@webex/plugin-authorization-browser-first-party/test/unit/spec/authorization.js

```diff
@@ -180,11 +180,12 @@ browserOnly(describe)('plugin-authorization-browser-first-party', () => {
       });
       describe('when the url contains an error', () => {
         it('throws a grant error', () => {
-          assert.throws(() => {
-            makeWebex(
-              'http://127.0.0.1:8000/?error=invalid_scope&error_description=The%20requested%20scope%20is%20invalid.'
-            );
-          }, /The requested scope is invalid./);
+          try {
+            makeWebex('http://127.0.0.1:8000/?error=invalid_scope&error_description=The%20requested%20scope%20is%20invalid.');
```

Comment:
we are giving invalid url as argument to makeWebex function so it will throw every time regardless of the syntax. We just changed syntax to support jest as assert.throws was not supported as jest syntax.

Reply:
my point is that now if someone changes makeWebex and introduces a bug such that makeWebex doesn't throw when given invalid url then this test won't catch it and the whole point of existence for this test is to catch such bug. The expect() is only called if error is thrown. If makeWebex doesn't throw, the test will pass and it should not pass. You need to modify the test so that it fails if makeWebex doesn't throw.

Reply:
Got it. Changed the syntax. Now this test will fail if `makeWebex()` doesn't throw.